### PR TITLE
feat: navigation feature

### PR DESCRIPTION
**Pull request by ankit9015**
Navigation feature includes:
1. Navigation bar
2. Navigation bar is responsive as per the viewport width

https://user-images.githubusercontent.com/67504902/154802887-c1b42474-66b4-4608-aa49-56533f199670.mp4

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Good work! you can merge

File: components/badge/badge.css [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 
 .badge-count {
-    font-size: 1em;
+    font-size: 0.5em;
     padding: 0.1rem 0.35rem;
     border-radius: 50%;
     background-color: rgb(0, 255, 76);
```
Comment (sushil-mod) [inline components/badge/badge.css:14]: if possible use color variable

**Review by sushil-mod — APPROVED** [review]
just one comment ,good to go

**Review by harshalini — APPROVED** [review]
Code is clean, you can merge this

**Pull request merged**